admin/auth_token.go

```diff
@@ -187,9 +187,17 @@ func (s *Service) IssueMagicAuthToken(ctx context.Context, opts *IssueMagicAuthT
 		expiresOn = &t
 	}
 
+	// Encrypt the secret
+	// TODO: revisit this
+	encryptedSecret, err := tkn.EncryptSecret([]byte(tkn.ID.String()))
```

Comment:
Good job getting started on this issue! About column-level encryption, there's a few things to think through. 

We also need column-level encryption for credentials, which is something we have been putting off, but have committed to building soon. So ideally we take an approach that we can also leverage there (to keep only one set of encryption keys for column-level encryption).

For the encryption keys, we would need those to be externally configured (e.g. with an environment variable or key file) and we need to support key rotation for them, which may entail persisting a key ID alongside the encrypted value.

There is also a question of where in the stack we handle this – one option is where the value is used, as you have done in this PR, but another is to handle it when values are read/written to the DB in the `admin/database/postgres` package, or using something like the `pgcrypto` Postgres extension and handling it in the DB itself.

Reply:
Sgtm! Closing this PR for now.